Assignment6DRY/src/assignment6dry/Assignment6DRY.java

```diff
@@ -0,0 +1,108 @@
+package assignment6dry;
+
+import java.util.Scanner;
+
+/**
+ * This program lets the user enter address and label sizing information to print
+ * up to 10 labels. 
+ * @author Samantha Shurie
+ */
+public class Assignment6DRY {
+
+    public static void main(String[] args) {
+       
+        Scanner keyboard = new Scanner(System.in);  //accepts input
+        int numLabels;  //stores the number of labels the user wishes to print
+        
+        //allow user to enter the number of labels they would like to print
+        println("Please enter the number of labels you wish to print:");
+        numLabels = keyboard.nextInt();
+        
+        //user can only enter up to 10 labels for this program
+        while(numLabels < 1 || numLabels > 10) {
+            println("Please enter a number between 1 and 10:");
+            numLabels = keyboard.nextInt();
+        }
+        
```

Comment:
I love how you changed this and made it so the user can enter in the amount of labels. Very nice!